packages/amazon/src/serverGroup/details/scalingPolicy/upsert/ScalingPolicyAdditionalSettings.tsx

```diff
@@ -0,0 +1,123 @@
+import { module } from 'angular';
+import { set } from 'lodash';
+import * as React from 'react';
+import { react2angular } from 'react2angular';
+
+import { HelpField, NumberInput, withErrorBoundary } from '@spinnaker/core';
+
+import { IUpsertScalingPolicyCommand } from '../ScalingPolicyWriter';
+
+export interface IScalingPolicyAdditionalSettingsProps {
+  command: IUpsertScalingPolicyCommand;
+  isInstanceType: boolean;
+  isNew: boolean;
+  operator: string;
+  updateCommand: (command: IUpsertScalingPolicyCommand) => void;
+}
+
+export const ScalingPolicyAdditionalSettings = ({
+  command,
+  isInstanceType,
+  isNew,
+  operator,
+  updateCommand,
+}: IScalingPolicyAdditionalSettingsProps) => {
+  const [commandView, setCommandView] = React.useState<IUpsertScalingPolicyCommand>(command);
+  const setCommandField = (path: string, value: number) => {
+    const newCommand = { ...command };
+    set(newCommand, path, value);
+    setCommandView(newCommand);
+    updateCommand(newCommand);
+  };
```

Comment:
Assume all this is necessary because this component is being used by an AngularJS component?  Did you try embedding a Formik + FormFields then updating the parent whenever any field changes?

Reply:
Yes, its part of the mid-migration dual-updates I had to do otherwise the react children were not re-rendering for 3-5 seconds after the command was updated and propagated through the angular controller. 

I haven't tried using Formik yet, but plan to at the high level of the modal. For your reference, my next PRs will be the parent modal for AWS and then Titus. I intend to make the modal the centralized place for the command and change handler. Once this happens, I can go remove some of this mid-migration tech debt. Here are my thoughts on what it would look like: 

1. Refactor parent modal to centalize command building and updating
2. Remove dual-updates/clones from children components. Remove references to a legacy observables that helped manage updates between angular and react. 
3. Incorporate `SpinFormik` and `FormikFormField`s to the modal sections.